ko/news/_posts/2024-05-30-datadog-oss-program.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: news_post
+title: "Datadog가 ruby-lang.org의 OSS 커뮤니티 지원을 제공합니다"
+author: "hsbt"
+translator: "shia"
+date: 2024-05-30 00:00:00 +0000
+lang: ko
+---
+
+우리는 Ruby의 공식 웹사이트인 ruby-lang.org가 모니터링을 위해 Datadog을 적용했다는 것을 알려드립니다. 이는 [Datadog OSS 커뮤니티 지원](https://opensource.datadoghq.com/projects/oss-program/)을 통해 이루어졌습니다.
```

Comment:
"Datadog" would be written as "데이터도그" in Korean, so let's use 는/를 instead of 은/을.

```suggestion
우리는 Ruby의 공식 웹사이트인 ruby-lang.org가 모니터링을 위해 Datadog를 적용했다는 것을 알려드립니다. 이는 [Datadog OSS 커뮤니티 지원](https://opensource.datadoghq.com/projects/oss-program/)을 통해 이루어졌습니다.
```

Reply:
+1 c434961